src/entities/applet/model/hooks/useStartEntity.ts

```diff
@@ -61,30 +75,57 @@ function useStartEntity() {
     );
   }
 
-  function startActivity(
+  async function startActivity(
     appletId: string,
     activityId: string,
     eventId: string,
-  ) {
-    return new Promise<boolean>(resolve => {
+    isTimerElapsed: boolean = false,
+  ): Promise<StartResult> {
+    const isOnline = await isAppOnline();
+
+    return new Promise<StartResult>(resolve => {
+      if (
+        !isOnline &&
+        lookupMedia({
+          appletId,
+          entityId: activityId,
+          entityType: 'regular',
+          queryClient,
+        })
```

Comment:
Please move the function call with the parameters to a separate function expression for more clarity